### ISSUE TITLE
GitHub Copilot Chat view cannot be scrolled using Page Up/Page Down keys, only mouse wheel works

### ISSUE THREAD

**Issue by Hunter6324** [microsoft/vscode#320066]
Type: <b>Bug</b>

**Steps to Reproduce:**
1.  Open Visual Studio Code and enable the GitHub Copilot extension.
2.  Open the GitHub Copilot Chat panel and start a new conversation by sending a query.
3.  After receiving a sufficiently long response that requires scrolling, attempt to navigate the chat history using the **Page Up** or **Page Down** keys on the keyboard.
4.  Observe that the chat view does not scroll when using these keys.
5.  Verify that scrolling works correctly when using the mouse wheel or by dragging the scrollbar directly.

**Expected Result:**
The GitHub Copilot Chat panel should allow scrolling up and down through the conversation history using the Page Up and Page Down keys, consistent with the behavior in other VS Code text editors and panels.

**Actual Result:**
The Page Up and Page Down keys have no effect on the chat view. Scrolling is only possible via mouse wheel or scrollbar drag.

VS Code version: Code 1.123.0 (6a44c352bd24569c417e530095901b649960f9f8, 2026-06-03T11:29:03+02:00)
OS version: Windows_NT x64 10.0.26200
Modes:
Remote OS version: Linux x64 6.6.114.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i5-12400 (12 x 2496)|
|GPU Status|2d_canvas: enabled<br>GPU0: VENDOR= 0x10de, DEVICE=0x2803 [NVIDIA GeForce RTX 4060 Ti], DRIVER_VENDOR=NVIDIA, DRIVER_VERSION=32.0.16.1047 *ACTIVE*<br>GPU1: VENDOR= 0x10de, DEVICE=0x2803 [NVIDIA GeForce RTX 4060 Ti], DRIVER_VERSION=32.0.16.1047<br>GPU2: VENDOR= 0x1414, DEVICE=0x008c [Microsoft Basic Render Driver], DRIVER_VERSION=10.0.26100.8521<br>Machine model name: <br>Machine model version: <br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|63.75GB (25.71GB free)|
|Process Argv|-n --crash-reporter-id c66132d5-1845-4b05-84a5-dd242e12e1d0|
|Screen Reader|yes|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-24.04|
|OS|Linux x64 6.6.114.1-microsoft-standard-WSL2|
|CPUs|12th Gen Intel(R) Core(TM) i5-12400 (8 x 0)|
|Memory (System)|31.35GB (28.28GB free)|
|VM|0%|
</details><details><summary>Extensions (14)</summary>

Name|Identifier|Author|Version
---|---|---|---
esbuild Problem Matchers|connor4312.esbuild-problem-matchers|connor4312|0.0.3
GitHub Codespaces|github.codespaces|GitHub|1.18.13
开发容器|ms-vscode-remote.remote-containers|ms-vscode-remote|0.459.0
Remote - SSH|ms-vscode-remote.remote-ssh|ms-vscode-remote|0.124.0
Remote - SSH: Editing Configuration Files|ms-vscode-remote.remote-ssh-edit|ms-vscode-remote|0.87.0
WSL|ms-vscode-remote.remote-wsl|ms-vscode-remote|0.104.3
远程开发|ms-vscode-remote.vscode-remote-extensionpack|ms-vscode-remote|0.26.0
远程资源管理器|ms-vscode.remote-explorer|ms-vscode|0.5.0
Remote - Tunnels|ms-vscode.remote-server|ms-vscode|1.5.3
VS Code Speech|ms-vscode.vscode-speech|ms-vscode|0.16.0
Chinese (Simplified, China) language support for VS Code Speech|ms-vscode.vscode-speech-language-pack-zh-cn|ms-vscode|0.5.1
open in browser|techer.open-in-browser|techer|2.0.0
vscode-pdf|tomoki1207.pdf|tomoki1207|1.2.2
Chinese (Simplified) (简体中文) Language Pack for Visual Studio Code|ms-ceintl.vscode-language-pack-zh-hans|MS-CEINTL|1.123.2026060414

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
binariesv615:30325510
nativeloc1:31344060
dwcopilot:31170013
dwoutputs:31242946
copilot_t_ci:31333650
e5gg6876:31282496
pythonrdcb7:31342333
6518g693:31463988
aj953862:31281341
6abeh943:31336334
envsactivate1:31505457
cloudbuttont:31379625
3efgi100_wstrepl:31403338
ddidt:31399633
ec5jj548:31422691
cp_cls_t_966_ss:31526232
inlinechat_v2_hd992725:31505030
4je02754:31466945
8hhj4413:31478653
ge8j1254_inline_auto_hint_haiku:31490510
38bie571_auto:31478678
cp_cls_c_1081:31454833
conptydll_true:31498968
ia-use-proxy-models-svc:31452481
e9c30283:31461165
test_treatment2:31471001
46204921:31447328
idci7584:31464702
e3e4d672:31494082
ei9d7968:31496641
nes-extended-on:31455476
chat:31457767
8hig5102:31480529
89g7j272:31518289
7e187181:31503455
i2gc6536:31499202
52612955:31516516
ghj88844:31499326
h08i8180:31475369
ddid_c:31478207
hmra_i5g22:31518061
getcmakediagnosticson:31489824
7df3h592:31512476
pro_large_t:31499376
cp_cls_t_1082:31516087
logging_enabled_new:31498466
j0d79568:31499440
jb_cp_cls_c_632:31524806
748c7209:31512887
32d76977:31512328
ha629193:31508444
cdk-lw-on:31524445
nes-charagree-0:31521369
31fi7170_t:31522396
a1ije391_t:31515488
api_cot_ctrl:31513533
g4a0e106:31517209
95db8703:31523165
61138546:31518536
d7b18187:31526828

```

</details>
